cmd/state-installer/installer.go

```diff
@@ -97,6 +97,11 @@ func (i *Installer) Install() (rerr error) {
 		return errs.Wrap(err, "Could not update PATH")
 	}
 
+	err = i.cfg.Set(constants.InstalledAdminConfig, isAdmin)
+	if err != nil {
+		return errs.Wrap(err, "Failed to set current privilege level in config")
+	}
```

Comment:
```suggestion
	err = i.cfg.Set(constants.InstalledAdminConfig, isAdmin)
	if err != nil {
		return errs.Wrap(err, "Failed to set current privilege level in config")
	}
```

This doesn't seem viable since we expose `state clean config` and `state clean uninstall` as separate steps. Meaning you can clean your config, which would purge this value, and the run uninstall which would no longer have knowledge over how it was installed.

Since this is Windows only I think either we save it to the registry, or potentially we could just check the ownership on the state.exe?

Reply:
Checking the ownership of the file requires us to interact with the Windows ACL so I opted to store this in the registry.